index.html

```diff
@@ -1,15 +1,43 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width" />
-    <title>replit</title>
-    <link href="style.css" rel="stylesheet" type="text/css" />
-    <link href="profile.css" rel="stylesheet" type="text/css" />
-  </head>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>replit</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link href="profile.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <aside> sider bar here </aside>
+  <main>
+    <form action="">
+      <div class="container">
+        <div class="heading">
+          <h1>Fill in your profile form details</h1>
+          <p>Kindly ensure all details given are correct</p>
+        </div>
+        <div class="row">
+          <div class="col">
```

Comment:
not sure how this will turn out, but I think using two `col` class and each have 4 elements can cause unintentional bug, on mobile.

Reply:
Should I adjust it?